awebox/tools/print_operations.py

```diff
@@ -77,8 +77,8 @@ def hline(charact):
 def get_awebox_license_info():
     license_info = []
     license_info += [hline('+')]
-    license_info += ['This is awebox, a modeling and optimization framework for multi-kite AWE systems..']
-    license_info += ['awebox is free software; you can redistribute it and/or modify it under the terms']
+    license_info += ['This is the awebox, a modeling and optimization framework for multi-kite AWE systems..']
+    license_info += ['The awebox is a free software; you can redistribute it and/or modify it under the terms']
```

Comment:
Regarding software branding, there are two software package names going around:

- "`awebox`"
- "The `awebox`"

I would argue to call the software simply `awebox` (one would also not use  "the `CasADi`" or "the `acados`", "the `Python`) and adapt the license info displayed again accordingly.

Reply:
Hi Jochem, 

1. I'll remove the pdb's (oops.) 

2. The only changes to the steady state actuator disk model are in opts.default, ie, how the options-tree labels are named. 

3. Almost all of the variables are lifted... But, there are some states that are associated with the unsteady model. I'm not opposed to making things more specific again (so that the steady state model only has lifted variables), but I really really want to merge before the differences become huge.

4. A "box" is a noun - so it has an article. Since the awebox is a "box" related to "AWE", it should be called "the awebox". Since neither casadi, or acados refer to existing nouns, that's not true for them... 
That is, it sounds really really weird to me to say "doing xxxxxx in awebox". 
---> (mach es in Schachtel)(doe het in doos) would be weird, right?

Reply:
1. Cheers!

2. Great, I just wanted to know if some of the steady-state equations changed in case I re-use the model in the future.

3. Works for me, I just have the compatibility with the DAE framework in mind (so that you can use CasADi integrators for simulating the system etc..). But we can deal with it later.

4. In this context, `awebox` is a name not a common noun, hence without article (see also matlab, matplotlib,...). I see the friction when using it in sentences, but then we need another name. @thilobro thoughts?